test/ex_machina_test.exs

```diff
@@ -98,4 +98,23 @@ defmodule ExMachinaTest do
     }
     assert records == [expected_record, expected_record, expected_record]
   end
+
+  test "create/2 raises helpful error denoting use of insert/2" do
+    assert_raise RuntimeError, ~r/create\/2 has been removed/, fn ->
+      Factory.create(:user, admin: true)
+    end
+  end
+
+  test "create_pair/2 raises helpful error denoting use of insert_pair/2" do
+    assert_raise RuntimeError, ~r/create_pair\/2 has been removed/, fn ->
+      Factory.create_pair(:user, admin: true)
+    end
+  end
+
+  test "create_list/3 raises helpful error denoting use of insert_list/3" do
+    assert_raise RuntimeError, ~r/create_list\/3 has been removed/, fn ->
+      Factory.create_list(3, :user, admin: true)
+    end
+  end
```

Comment:
Yep! That looks good
> On May 11, 2016, at 10:17 AM, David Kuhta <notifications@github.com> wrote:
> 
> In test/ex_machina_test.exs <https://github.com/thoughtbot/ex_machina/pull/129#discussion_r62856047>:
> 
> > +    assert_raise RuntimeError, ~r/create\/2 has been removed/, fn ->
> > +      Factory.create(:user, admin: true)
> > +    end
> > +  end
> > +
> > +  test "create_pair/2 raises helpful error denoting use of insert_pair/2" do
> > +    assert_raise RuntimeError, ~r/create_pair\/2 has been removed/, fn ->
> > +      Factory.create_pair(:user, admin: true)
> > +    end
> > +  end
> > +
> > +  test "create_list/3 raises helpful error denoting use of insert_list/3" do
> > +    assert_raise RuntimeError, ~r/create_list\/3 has been removed/, fn ->
> > +      Factory.create_list(3, :user, admin: true)
> > +    end
> > +  end
> Sure, so something like:
> 
>   test "raises helpful errors when using the old create functions" do
>     assert_raise RuntimeError, ~r/create\/2 has been removed/, fn ->
>       Factory.create(:user, admin: true)
>     end
> 
>     assert_raise RuntimeError, ~r/create_pair\/2 has been removed/, fn ->
>       Factory.create_pair(:user, admin: true)
>     end
> 
>     assert_raise RuntimeError, ~r/create_list\/3 has been removed/, fn ->
>       Factory.create_list(3, :user, admin: true)
>     end
>   end
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/ex_machina/pull/129/files/8ee1e519cc97978edb17218bc2a7eb75046dff0d#r62856047>